src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/FinalLocalVariable/context/defaultContext.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Test.java b/Test.java
+index 2a731db..eb996ad 100644
+--- a/Test.java
++++ b/Test.java
+@@ -2,12 +2,12 @@ package TreeWalker.coding.FinalLocalVariable;
+ 
+ public class Test {
```

Comment:
please add example with local variable on multiple lines:
```java
int myconst
    = calculateConst(1,2);
```
```diff
int myconst
-    = calculateConst(9, 2);
+    = calculateConst(1, 2);
```
Context strategy should handle this case.

Reply:
done